docs/tutorials/want-to-use-python.md

```diff
@@ -274,6 +274,40 @@ Returns the hash value of this object, internally it calls the CPython's [`PyObj
 
 Returns a corresponding primitive value for this object, see [`Symbol.toPrimitive` on MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Symbol/toPrimitive) for more details.
 
+#### With ES-Modules
```

Comment:
```suggestion
## Working with ECMAScript Modules
```
I read the entire document and feel that it should be placed in the level 2 :)

Reply:
done